src/config/theme.js

```diff
@@ -0,0 +1,23 @@
+import { createMuiTheme } from 'material-ui/styles';
+import indigo from 'material-ui/colors/indigo';
+import pink from 'material-ui/colors/pink';
+import red from 'material-ui/colors/red';
+
+// All the following keys are optional.
+// We try our best to provide a great default value.
+const bodhiTheme = createMuiTheme({
+  palette: {
+    primary: indigo,
+    secondary: pink,
```

Comment:
is this used anywhere yet?

Reply:
not yet but this is going to be used in the next PR